### DISCUSSION TITLE
Received a notice: `The VundleVim organization has been flagged`

### DISCUSSION THREAD

**Discussion by gmarik** [community/community#48173] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Received a notice that:
> The VundleVim organization has been flagged.
Because of that, your organization is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your organization’s status reviewed.

How do I resolve the issue? Thank you!

**Comment by gmarik** [comment, marked as answer]
Moved to support.

**Reply by iamrakhmatov** [reply]
Hey, I'm currently facing the same issue, how can I solve this? could you find a solution?

**Reply by SemenMartynov** [reply]
The same.
I see three discussions with this issue on Hacker News and no one understands what's going on.

**Reply by jplante01** [reply]
I started a ticket: 

"The vundle.vim repository by the VundleVim organization is a package the my workflow relies on... To have it disappear with no explanation is concerning. It has been gone for two days, please reinstate VundleVim's privileges asap, or at the very least provide an explanation for this action."

**Reply by pave-reece** [reply]
I also cannot access this and it is my go to for my Vim setup.

**Reply by jplante01** [reply]
Has the Github staff provided an explanation yet? I received a 'we're looking into it' response, yuck.

**Reply by superstarlancer** [reply]
I cannot access this and it is my go to for my Vim setup too.

**Comment by bradfeehan** [comment]
This repository disappearing has broken things for me, it's legitimate, please reinstate <3

**Reply by VibroAxe** [reply]
> This repository disappearing has broken things for me, it's legitimate, please reinstate <3

Agreed

**Reply by 515k4** [reply]
Same for me.

**Reply by ruhnet** [reply]
Same!

**Reply by RichardT2023** [reply]
Same here. Please reinstate this account.

**Reply by JarredAllen** [reply]
Same here!

**Reply by ratonatiuh** [reply]
same for me

**Comment by hbto** [comment]
Hello, I am affected by the disappearing of this project too.

Is there a reason for its disappearance?

Regards.

**Comment by rds13** [comment]
Please do restore access to VundleVim or give us some explanations.

**Reply by Stratus3D** [reply]
I'm affected by the disappearance of this repo as well. Please provide more information as to why it is hidden.

**Comment by Stratus3D** [comment]
Related: https://news.ycombinator.com/item?id=34885269

**Reply by Stratus3D** [reply]
I have pushed up an old local copy of the repo to my personal account so I can work around the issue: https://github.com/Stratus3D/Vundle.vim

**Comment by jplante01** [comment]
This is valid package, and its disappearance is affecting my workflow. For it to be missing for this long without an explanation is concerning.

**Comment by alanbrent** [comment]
Looks like the `Vundle.vim` repo had a publicly editable wiki, which had been updated to link to malware. I suspect this is why the account was taken offline.

https://news.ycombinator.com/item?id=34194055
https://old.reddit.com/r/vim/comments/zzcn10/vimorg_autosave_vundle_plugin_windows_installer/

**Comment by francescoboc** [comment]
What's going on?

**Comment by ruhnet** [comment]
The slowness of GitHub to resolve this (or at least provide a reasonable explanation) with such a prominent repo casts aspersions on their credibility/reliability for serious projects.

**Comment by csr13** [comment]
Yooo!!!

**Comment by doc75** [comment]
This might the sign that Github is not a safe place for such opensource projects as there is no way to have real human contact in case of issue.

**Reply by sk3l** [reply]
+1000

**Comment by sk3l** [comment]
I am learning a valuable lesson from this event: **it is no longer safe/resilient to have your dev processes rely on GitHub**.

No idea what is causing this (let's call it what it is) outage in serving a key public repository for devs that need this package manager to bootstrap work environments.

If the root cause was a publicly editable Wiki that was pointing to malware...then please just clean the Wiki and lock it down, don't nuke the entire repo and the containing user/org (!?).

**Comment by rokkitlawnchair** [comment]
Hello,

may I suggest https://codeberg.org/ as a backup/mirror repository/platform?

**Reply by doc75** [reply]
Having several mirror seems to be the default rule to apply to avoid such outage.

**Comment by aronwolf90** [comment]
Same problem here. Pleas solve this.

**Comment by roolebo** [comment]
@github-staff anyone there? Please look into this issue.

**Comment by essial** [comment]
This is why I gave up on open source projects that aren't owned/fully backed by commercial companies. Why anyone would think to leave their wiki open to public editing without an extremely active moderation team is beyond me.

**Reply by mythos43** [reply]
Hypocritical much?

**Reply by JarredAllen** [reply]
This is why I gave up on commercial projects that aren't owned/fully backed by open-source projects. Why anyone would think to trust a large corporation without having the ability to fork the project if they abuse their power is beyond me.

**Comment by user5060** [comment]
my account has been flagged, i did not violate any rules, **please unflagged/restore my account**

**Comment by aneichyk** [comment]
@github-staff  please address the issue asap!

**Comment by justonia** [comment]
This is f'n ridiculous. A repo that hasn't changed in a decade is suddenly flagged and removed?

**Comment by jonbrouse** [comment]
Will there be any type of security disclosure around this?

**Comment by cdcooksey** [comment]
@github-staff Please reinstate asap

**Comment by estebanz01** [comment]
It seems it's back again: https://github.com/vundlevim
